lib/vk_funcs.c

```diff
@@ -164,13 +164,30 @@ ktxLoadVulkanLibrary(void)
     }
 #endif
 
-#include "vk_funclist.inl"
+// #include "vk_funclist.inl"
```

Comment:
I think the VK_FUNCTION definitions above, lines 84-117 can be deleted.

Reply:
Thanks for the catch, I'll fix that right away 👍